### PR TITLE
Features/98

### PR DESCRIPTION
**Pull request by sorobon**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

It would be a great feature add support for templates in collections.
I want to use ansible vars in my collection files and now its impossible.

### Applicable Issues

[#98 ]

### CODE REVIEW

File: molecule/setup_with_collections/templates/collections/sample_techproducts_configs/solrconfig.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
        that you fully re-index after changing this setting as it can
        affect both how text is indexed and queried.
   -->
-  <luceneMatchVersion>8.0.0</luceneMatchVersion>
+  <luceneMatchVersion>{{ solr_version }}</luceneMatchVersion>
```
Comment (dortegau) [inline molecule/setup_with_collections/templates/collections/sample_techproducts_configs/solrconfig.xml]: I would prefer to create a new variable called `solr_lucene_match_version` and set `solr_version` to that variable as default value. I think that is common to have lucene match version under solr version used.

File: defaults/main.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 
 ## General
-solr_version: 8.2.0
+solr_version: 8.3.0
```
Comment (dortegau) [inline defaults/main.yml]: This change is not reflected in changelog :p

Comment (sorobon) [reply defaults/main.yml]: done

File: molecule/setup_with_collections/templates/collections/sample_techproducts_configs/solrconfig.xml (hunk 1 above)
Comment (sorobon) [reply molecule/setup_with_collections/templates/collections/sample_techproducts_configs/solrconfig.xml]: It's an example.

**Pull request merged**